lumicks/pylake/scan.py

```diff
@@ -381,10 +393,17 @@ def export_video_rgb(
         **kwargs
             Forwarded to :func:`matplotlib.pyplot.imshow`.
         """
-        self._export_video(
+        self.export_video(
```

Comment:
Yeah, it's a good point, since numerical values are not informative on the call side. I will amend it for `export_video` so that in the future we'll have a call with kwargs for those.

I wouldn't change the deprecated ones though, because introducing a breaking API change in an API that we are already deprecating doesn't make that much sense to me.